nservicebus/rabbitmq/routing-topology_support-delayed-delivery_rabbit_[4,].partial.md

```diff
@@ -0,0 +1,7 @@
+### Delayed delivery
+
+Starting with Version 4.3, it is possible to support [delayed delivery](delayed-delivery.md) without requiring the [timeout manager](/nservicebus/messaging/timeout-manager.md) by implementing `ISupportDelayedDelivery` in addition to `IRoutingTopology` in the routing topology class.
```

Comment:
> Starting with Version 4.3

In Versions 4.3 and above, 

> it is possible to support [delayed delivery](delayed-delivery.md)

RabbitMQ supports [delayed delivery](delayed-delivery.md) ...

> by implementing `ISupportDelayedDelivery` in addition to `IRoutingTopology` in the routing topology class.

... and further text

Question (not sure what's the answer) - is that level of detail relevant for the reader? or do we go too much into implementation details?